deploy/env/dev.yaml

```diff
@@ -11,4 +11,6 @@ registration-service:
                   "resource": "crt",
                   "clientId": "crt",
                   "public-client": true
-                }'
\ No newline at end of file
+                }'
+host-operator:
+  duration-before-change-request-deletion: '1m'
```

Comment:
I have two reasons:
1. I needed the short duration for the demo on Wednesday :-) 
2. I was thinking if the long duration makes any sense for the dev environment because:
    * we deploy the dev environment on 10h clusters
    * when we use the dev, we usually want to try it out if it works as expected. In case I want to try the deletion of the ChangeTierRequest, then I don't want to wait for 24h for that :-)

Reply:
These are valid reasons but... When we have some UI (admin console or something) which uses these ChangeTierRequiests then one minute timeout could be too short. Also, having dev env as close to prod as possible is almost always a good idea.
We have tests for short timeouts so we don't necessarily need to keep different timeouts in dev and prod for the testing purposes.

Reply:
fair enough - removed in 732d4ed